.github/workflows/deploy.yml

```diff
@@ -75,6 +75,46 @@ jobs:
           package: ./bin/wasm-node/javascript/package.json
           access: public
 
+  deno-publish:
+    runs-on: ubuntu-latest
+    # This action checks if a certain git tag exists. If not, it compiles the JavaScript package,
+    # then commits the compilation artifacts, tags the commit, and pushes the tag.
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0   # Necessary below for checking if the tag exists.
+      - uses: actions/setup-node@v3.4.1
+        with:
+          node-version: 14
```

Comment:
didn't you change it to 12 recently?

Reply:
I haven't finished the PR: https://github.com/paritytech/smoldot/pull/2517